src/Component/Symbolizer/RasterChannelEditor/RasterChannelEditor.tsx

```diff
@@ -139,24 +103,35 @@ export class RasterChannelEditor extends React.Component<RasterChannelEditorProp
   /**
    * Checks if ChannelSelection is of type GrayChannel.
    */
-  isGrayChannel = (channels: ChannelSelection): channels is GrayChannel => {
+  // TODO: move to geostyler-style
```

Comment:
Could you create a ticket for this, so we do not forget about it?

Reply:
https://github.com/geostyler/geostyler-style/pull/393